simulations/vip-502/basesepolia.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, pretendExecutingVip, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import { PSR } from "../../multisig/proposals/basesepolia/vip-003";
+import vip007 from "../../multisig/proposals/basesepolia/vip-007";
+import { PLP, PRIME } from "../../multisig/proposals/basesepolia/vip-007";
+import vip502, {
+  BOUND_VALIDATOR,
+  COMPTROLLERS,
+  VTOKENS,
+  XVS_BRIDGE,
+  XVS_BRIDGE_ADMIN,
+  XVS_STORE,
+} from "../../vips/vip-502/bsctestnet";
```

Comment:
I would import every address from `../../multisig/proposals/basesepolia/vip-007`, to have a single point of truth